docs/doxygen/doxygen_wrapper.py

```diff
@@ -18,6 +20,11 @@
 
 
 class DoxygenWrapper(object):
+    SVG_IMAGE_PATTERN = re.compile(
+        """ src="(?P<svg_filename>[a-z0-9_]+\\.svg)" """
```

Comment:
are all filenames guaranteed to be lower case? no other weird chars besides _?

Reply:
Yes, at least at this point from doxygen.
The key is to stay consistent and don't break pages (i.e. if file is not embedded - it won't get deleted).